notebooks/20_markov_decision_processes_part_2/index.md

```diff
@@ -0,0 +1,672 @@
+<div style="direction:ltr;line-height:300%;">
+	<font face="XB Niloofar" size=20>
+        Markov Decision (from value iteration)
+    </font>
+    </div>
+
+<br/>
+
+<br/>
+
+<h1> Table of Contents </h1>
+---
```

Comment:
Hi,
Well done with your Lecture Note!
It is complete and comprehensive.
I wrote down a few comments on different parts. Modify your entire file according to these notes.
I believe you need to revise the writing language in order to enhance it to the top level.
Grammar, punctuation, and proper collocations should be considered.
Furthermore, after reviewing your note on my vs code, some of the images were too close to the next subject (specifically at the end of the LN).
Take a look at this issue as well.

Good Luck!